src/main/java/kitchenpos/domain/Menu.java

```diff
@@ -10,6 +10,21 @@ public class Menu {
     private Long menuGroupId;
     private List<MenuProduct> menuProducts;
 
+    public Menu() {
```

Comment:
```suggestion
    public Menu() {}
```
이 부분은 중요한 것은 아니지만 [컨벤션](https://naver.github.io/hackday-conventions-java/)에 따르면 이렇게 써줄 수도 있을 것 같아요...!